.pre-commit-hooks.yaml

```diff
@@ -18,6 +18,22 @@
   types:
       - text
 
+- id: codespell-complete
+  entry: >
+      codespell --builtin clear,en-GB_to_en-US,informal,name,rare,usage --check-filenames
```

Comment:
Why not put this in `pyproject.toml` config, so CLI users can get it too?

Reply:
Added